tests/robot-tests/tests/admin_and_public/bau/legacy_releases.robot

```diff
@@ -33,36 +38,86 @@ Create legacy release
     user clicks button    OK
     user waits until modal is not visible    Create legacy release
     user waits until h1 is visible    Create legacy release
-    user enters text into element    id:legacyReleaseForm-description    Test collection
+    user enters text into element    id:legacyReleaseForm-description    ${DESCRIPTION}
     user enters text into element    id:legacyReleaseForm-url    http://test.com
     user clicks button    Save legacy release
 
 Validate created legacy release
     user waits until h1 is visible    Manage publication
     user checks element count is x    css:tbody tr    1
     user checks results table cell contains    1    1    1
-    user checks results table cell contains    1    2    Test collection
+    user checks results table cell contains    1    2    ${DESCRIPTION}
     user checks results table cell contains    1    3    http://test.com
 
+Navigate to admin dashboard to create new release
+    user navigates to admin dashboard    Bau1
+
+Navigate to release
+    user navigates to editable release summary from admin dashboard    ${PUBLICATION_NAME}
+    ...    Academic Year 2020/21 (not Live)
+
+Approve release
+    user clicks link    Sign off
+    user approves original release for immediate publication
+    user waits until page contains element    testid:public-release-url
+    ${PUBLIC_RELEASE_LINK}=    Get Value    xpath://*[@data-testid="public-release-url"]
+    check that variable is not empty    PUBLIC_RELEASE_LINK    ${PUBLIC_RELEASE_LINK}
+    Set Suite Variable    ${PUBLIC_RELEASE_LINK}
+
+Check legacy release appears on public frontend
+    user navigates to public frontend    ${PUBLIC_RELEASE_LINK}
+
+    user clicks element    testid:Expand Details Section See other releases (1)
+
+    user checks page contains element    //*[@data-testid="other-release-item"]//a[text()="${DESCRIPTION}"]
+
+    user checks element attribute value should be
+    ...    //*[@data-testid="other-release-item"]//a[text()="${DESCRIPTION}"]    href    http://test.com/
+
+Navigate to admin dashboard to update legacy release
```

Comment:
Seems like `Navigate to publication to update legacy releases` would be more appropriate